template/.github/workflows/main.js.yml.ejs

```diff
@@ -140,3 +158,10 @@ jobs:
           # SET THIS OR YOU WILL DEPLOY TO YOUR PERSONAL VERCEL NAMESPACE
```

Comment:
Is it clear what property we need to set? Can we add an indicator to make sure they understand which line above or below we are indicating with this comment.
```suggestion
          # SET THIS👇 OR YOU WILL DEPLOY TO YOUR PERSONAL VERCEL NAMESPACE
```

Reply:
good point. we should also set a followup to bring this into the CLI prompts when choosing Vercel